x-pack/legacy/plugins/siem/public/components/timeline/styles.tsx

```diff
@@ -57,10 +58,19 @@ TimelineBody.displayName = 'TimelineBody';
  * EVENTS TABLE
  */
 
-export const EventsTable = styled.div.attrs(({ className = '' }) => ({
-  className: `siemEventsTable ${className}`,
-  role: 'table',
-}))``;
+interface EventsTableProps {
+  columnWidths: number;
+}
+
+export const EventsTable = styled.div.attrs<EventsTableProps>(
+  ({ className = '', columnWidths }) => ({
+    className: `siemEventsTable ${className}`,
+    role: 'table',
+    style: {
+      minWidth: `${columnWidths}px`,
+    },
+  })
+)<EventsTableProps>``;
```

Comment:
are the backtics required here?

Reply:
yes, it's required by `styled-components`